Gemfile

```diff
@@ -93,3 +93,5 @@ gem 'dockerfile-rails', '>= 1.6', :group => :development
 gem 'redcarpet', '~> 3.6'
 
 gem "aws-sdk-s3", "~> 1.160", :require => false
+
+gem "rbnacl", "~> 7.1"
```

Comment:
what is this gem being used for? I think it's worth to start adding comments on top of new gems so we have context on why we need  them, otherwise we'll not remember it and to be honest I don't know this one or `hashie` ahaha.